### ISSUE TITLE
UK: Recovered cases data is unavailable

### ISSUE THREAD

**Issue by lisphilar** [lisphilar/covid19-sir#340]
## Summary
The number of recovered cases in United Kingdom is collected via COVID-19 Data Hub, but the number is much less than expected.

## (Optional) Related classes
- `covsirphy.DataLoader`
- `covsirphy.Scenario`

## Codes and outputs:
```Python
import covsirphy as cs
# Dataset preparation
data_loader = cs.DataLoader("input")
jhu_data = data_loader.jhu()
population_data = data_loader.population()
# Scenario analysis
snl = cs.Scenario(jhu_data, population_data, "UK")
# Show the records without complement
snl.complement_reverse().records()
```
![index](https://user-images.githubusercontent.com/7270139/99245233-99e9f300-2846-11eb-9c6b-0f94fe0ec413.png)

|     | Date                |   Confirmed |   Infected |   Fatal |   Recovered |
|----:|:--------------------|------------:|-----------:|--------:|------------:|
| 274 | 2020-11-12 00:00:00 |     2436277 |    2337888 |   95358 |        3031 |
| 275 | 2020-11-13 00:00:00 |     2477386 |    2378281 |   96016 |        3089 |
| 276 | 2020-11-14 00:00:00 |     2504946 |    2404971 |   96867 |        3108 |
| 277 | 2020-11-15 00:00:00 |     2529923 |    2429623 |   97179 |        3121 |
| 278 | 2020-11-16 00:00:00 |     2529923 |    2429623 |   97179 |        3121 |

https://gist.github.com/lisphilar/bdf6acaa2329eee3ce2d8487a4fdf5c9#file-uk_202021116-ipynb

## Environment
- CovsirPhy version: 2.11.0
- Python version; 3.8
- Installation: pipenv
- System: Google Colaboratory

**Comment by lisphilar** [comment]
<worldometers: Recovered is untracked>
https://www.worldometers.info/coronavirus/country/uk/

<corona tracker: 135 cases as-of 16Nov2020>
https://www.coronatracker.com/ja/country/united-kingdom

I tried to get the number of recovered cases from linelist (`cs.DataLoader.linelist()` #281), but no recovered cases were included.
https://gist.github.com/lisphilar/b8d54da82013431ca7e0205b4e98472f#file-uk_linelist_20201116-ipynb

**Comment by lisphilar** [comment]
This was included in version 2.11.0-iota with #353 .
